src/main/java/com/spotinst/sdkjava/enums/AzureDiskModeEnum.java

```diff
@@ -0,0 +1,48 @@
+package com.spotinst.sdkjava.enums;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public enum AzureDiskModeEnum {
```

Comment:
Why all enums not starting with AzureStatefulNode?

Reply:
updated names of all enum files to start with AzureStatefulNode